lib/Core/ExecutorUtil.cpp

```diff
@@ -33,6 +33,75 @@ using namespace llvm;
 
 namespace klee {
 
+  ref<klee::ConstantExpr> Executor::evalConstant(const Constant *c) {
```

Comment:
@ccadar Sorry ignore that comment. I misread the diff. I have it set to split which I misread.